core/src/main/java/hudson/model/operations/Operation.java

```diff
@@ -0,0 +1,48 @@
+package hudson.model.operations;
+
+import org.kohsuke.stapler.StaplerRequest;
+import org.kohsuke.stapler.StaplerResponse;
+
+/**
+ * Created by haswell on 8/7/17.
```

Comment:
If you create new API objects, there should be a meaningful Javadoc instead of the standard template. `@since TODO` would be useful